### ISSUE TITLE
Release Gumtree Spoon

### ISSUE THREAD

**Issue by GerardPaligot** [SpoonLabs/gumtree-spoon-ast-diff#16]
There is any plan to release this project on Maven Central?

**Comment by martinezmatias** [comment]
I would be great. It would simplify the installation of tools that use it such as Coming https://github.com/Spirals-Team/coming

**Comment by GerardPaligot** [comment]
To publish on Maven Central, we must:

1. Prepare the pom.xml file.
2. Make a request on Sonatype.
3. Publish.

To make a request on Sonatype, we communicate our group id and as soon as Maven Central open a way to release on this group id, we can make releases with any artifact id on this group id.

For now, we have:

- group id: `fr.inria.spirals`
- artifact id: `gumtree-spoon-ast-diff`

I don't care about the group id we'll choose but "spirals" is temporary. In some years, the team will have another name. So it isn't maybe the better solution. Have you an opinion about this point?

Another point is credential. To publish on maven central, we must sign our jar with GPG credential. How manage this credential?

**Comment by GerardPaligot** [comment]
I spoke with Lionel about this issue. We found 2 propositions: `fr.inria.gforge.spoon.labs` (subdomain of the Spoon group id) or the actual `fr.inria.spirals`.

What do you think about this?

**Comment by martinezmatias** [comment]
I'd go for r.inria.gforge.spoon.labs.

**Comment by GerardPaligot** [comment]
Congrats, Gumtree Spoon has been released and is on the way for Maven Central! :)

**Comment by martinezmatias** [comment]
Cool Gerard! Nice work!

**Comment by GerardPaligot** [comment]
```xml
<dependency>
    <groupId>fr.inria.gforge.spoon.labs</groupId>
    <artifactId>gumtree-spoon-ast-diff</artifactId>
    <version>1.0.0</version>
</dependency>
```

Available on Maven Central.

**Comment by monperrus** [comment]
FYI CD job fixed and gumtree-spoon-ast-diff 1.70 is published
https://repo1.maven.org/maven2/fr/inria/gforge/spoon/labs/gumtree-spoon-ast-diff/1.70/

**Comment by algomaster99** [comment]
We have releases now - https://mvnrepository.com/artifact/fr.inria.gforge.spoon.labs/gumtree-spoon-ast-diff.
